src/test/java/com/microsoft/sqlserver/jdbc/resultset/DataClassificationTest.java

```diff
@@ -10,13 +10,19 @@
 import java.sql.Statement;
```

Comment:
I think Azure now has DataClassification support. Could you remov Azure tags from tests and check if they pass?

Reply:
? yes it's supported and had been removed and tested. The only place that has these tags is in the negative test